source/riteway.js

```diff
@@ -1,25 +1,31 @@
 const tape = require('tape');
 
+const noop = new Function();
+
 // The testing library: a thin wrapper around tape
-const describe = (unit, cb) => tape(unit, assert => {
-  const end = assert.end.bind(assert);
-
-  const result = cb(description => ({
-    assert: ({ actual, expected, given, should = description }) =>
-      assert.same(
-        actual, expected,
-        `Given ${ given }: should ${ should }`
-      ),
-    // We can probably use async/await to deal with most async tests,
-    // but we should probably still supply `end()` so users can do
-    // whatever they like
-    end
-  }));
+const describe = (unit = '', cb = noop) => tape(unit, test => {
```

Comment:
Type hint is unnecessary here.

Reply:
Needed for accurate inferred types without TypeScript.